recipe/meta.yaml

```diff
@@ -1,33 +1,39 @@
-{% set opensensus_version = "0.7.10" %}
-{% set version = "0.1.2" %}
+{% set opencensus_version = "0.11.2" %}
+{% set version = "0.1.3" %}
 
 package:
   name: opencensus-context
   version: {{ version }}
 
 source:
-  url: https://github.com/census-instrumentation/opencensus-python/archive/v{{ opensensus_version }}.zip
-  sha256: 4ea0b5bac55c62f9ca5ddccc5acf8160c3575bf58bc5dab7bcb1f5a9cc578e71
+  url: https://github.com/census-instrumentation/opencensus-python/archive/v{{ opencensus_version }}.tar.gz
```

Comment:
Why use opencensus bundled software when you could use the standalone release tarball? https://files.pythonhosted.org/packages/4c/96/3b6f638f6275a8abbd45e582448723bffa29c1fb426721dedb5c72f7d056/opencensus-context-0.1.3.tar.gz

Is the vendored version a fork? If yes, the package name would need to reflect this.

Reply:
The `context` package is a part of the `opencensus` package, it's literally in the same tree.
https://github.com/census-instrumentation/opencensus-python/tree/master/context/opencensus-context
There is no reason to use one set of sources over the other, any such preference is purely personal.

Reply:
`opencensus`' setup.py itself pulls the code from pypi instead of fetching it from the full opencensus tarball: 

https://github.com/census-instrumentation/opencensus-python/blob/master/setup.py#L45

Reply:
Fixed.